Scenario/package.json

```diff
@@ -1,10 +1,10 @@
 {
     "name": "com.scenario.editor",
     "displayName": "Scenario Unity Plugin",
-    "version": "0.2.0",
+    "version": "0.2.3",
```

Comment:
No need to change also the version number, it is handled automatically

Reply:
this isn't the default package file so I don't believe it effects anything